server/views/serviceProviderReferrals/sessions/layoutExample.njk

```diff
@@ -0,0 +1,289 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
+
+{% extends "../partials/layoutAllComponents.njk" %}
+
+{% set pageTitle = "HMPPS Interventions" %}
+{% block pageTitle %}{{ pageTitle }} - GOV.UK{% endblock %}
+
+{% block primaryTitle %}
+    <h1 class="govuk-heading-l">This is a primary title example</h2>
+{% endblock %}
+
+
+{% block pageHeading %}
+  <h2 class="govuk-heading-l">Track progress of each intervention</h2>
+{% endblock %}
+
+{% block mainContent %}
```

Comment:
and here's `mainContent` (the original templates just use `content` but now we're adding a `mainContent` section and `sidebar`, as they need to both be part of the main page's content.